configure.ac

```diff
@@ -5830,7 +5831,9 @@ AC_ARG_WITH(
   [with_readline=readline]
 )
 
+
 AS_VAR_IF([with_readline], [readline], [
+  AX_CHECK_TYPEDEF(Function, readline/readline.h, [,AC_DEFINE([WITH_APPLE_EDITLINE]),])
```

Comment:
I am not sure that we have to add support for AS_VAR_IF([with_readline], [edit]) case too